tests/integration/lib/tabs.js

```diff
@@ -42,5 +39,11 @@ module.exports = function(client) {
 		about() {
 			return loadTab('about');
 		}
+	};
+
+	module.exports = function () {
```

Comment:
I don't think you can export a module from inside another module. What is meant to be happening here?